aea/helpers/ipfs/base.py

```diff
@@ -86,20 +87,21 @@ class IPFSHashOnly:
     # according to https://pkg.go.dev/github.com/ipfs/go-ipfs-chunker#pkg-constants
 
     @classmethod
-    def get(cls, file_path: str, wrap: bool = True) -> str:
+    def get(cls, file_path: str, wrap: bool = True, cid_v1: bool = True) -> str:
```

Comment:
We use the hash only tool in the most of the places where we deal with hashes, So I updated it to use cid v1 rather than updating all the top level APIs to use CID v1.